go/vt/sqlparser/ast_format.go

```diff
@@ -2577,6 +2577,101 @@ func (node *JSONUnquoteExpr) Format(buf *TrackedBuffer) {
 	buf.WriteString(")")
 }
 
+func (node *Count) Format(buf *TrackedBuffer) {
+	buf.astPrintf(node, "%s(", node.AggrName())
+	if node.Distinct {
+		buf.literal(DistinctStr)
+	}
+	buf.astPrintf(node, "%v)", node.Args)
+}
+
+func (node *CountStar) Format(buf *TrackedBuffer) {
+	buf.astPrintf(node, "%s(", node.AggrName())
```

Comment:
nit: just do a `buf.writeString("count(*)")`

Reply:
spoke offline . will remove it in next PR.